en/enable-tls-for-dm.md

```diff
@@ -0,0 +1,620 @@
+          duration: 8760h # 365d
+          renewBefore: 360h # 15d
+          organization:
+          - PingCAP
+          commonName: "TiDB"
+          usages:
+            - client auth
+          issuerRef:
+            name: ${cluster_name}-dm-issuer
+            kind: Issuer
+            group: cert-manager.io
+        ```
+
+        `${cluster_name}` is the name of the cluster. The above YAML file creates three objects:
+
+        - Set `spec.secretName` to `${cluster_name}-dm-master-cluster-secret`.
+        - Add `server auth` and `client auth` in `usages`.
+        - `dnsNames` and `ipAddresses` are not required.
+        - Add the Issuer created above in the `issuerRef`
+        - For other attributes, refer to [cert-manager API](https://cert-manager.io/docs/reference/api-docs/#cert-manager.io/v1alpha2.CertificateSpec)
+
+        After the object is created, `cert-manager` generates a `${cluster_name}-cluster-client-secret` Secret object to be used by the clients of the DM components.
+
+### Deploy the DM cluster
+
+When you deploy a DM cluster, you can enable TLS between DM components, and set the `cert-allowed-cn` configuration item to verify the CN (Common Name) of each component's certificate.
+
+> **Note:**
+>
+> Currently, you can set only one value for the `cert-allowed-cn` configuration item of DM-master. Therefore, the `commonName` of all `Certificate` objects must be the same.
+
+- Create the `dm-cluster.yaml` file:
+
+```yaml
+apiVersion: pingcap.com/v1alpha1
+kind: DMCluster
+metadata:
+  name: ${cluster_name}
+  namespace: ${namespace}
+spec:
+  tlsCluster:
+    enabled: true
+  version: v2.0.0-rc.2
+  pvReclaimPolicy: Retain
+  discovery:
+    address: "http://${tidb_cluster_name}-discovery.${tidb_namespace}:10261"
+  master:
+    baseImage: pingcap/dm
+    replicas: 1
+    storageSize: "1Gi"
+    config:
+      cert-allowed-cn:
+        - TiDB
+  worker:
+    baseImage: pingcap/dm
+    replicas: 1
+    storageSize: "1Gi"
+    config:
+      cert-allowed-cn:
+        - TiDB
+```
+
+Use the `kubectl apply -f dm-cluster.yaml` file to create a DM cluster.
+
+### Configure `dmctl` and connect to the cluster
+
+Get into the DM-master Pod:
+
+{{< copyable "shell-regular" >}}
+
+``` shell
+kubectl exec -it ${cluster_name}-dm-master-0 -n ${namespace} sh
+```
+
+Use `dmctl`:
+
+{{< copyable "shell-regular" >}}
+
+``` shell
+cd /var/lib/dm-master-tls
+/dmctl --ssl-ca=ca.crt --ssl-cert=tls.crt --ssl-key=tls.key --master-addr https://127.0.0.1:8261 list-member
+```
+
+## Use DM to migrate data between MySQL/TiDB databases that enable TLS for the MySQL client
+
+The following section describes how to configure DM to enable synchronously TLS-authenticated MySQL/TiDB database of MySQL client.
+
+To learn how to enable TLS for the MySQL client of TiDB, refer to [Enable TLS for the MySQL Client](enable-tls-for-mysql-client.md).
+
+### Step 1: Create the Kubernetes Secret object of TLS of each MySQL client
```

Comment:
```suggestion
### Step 1: Create the Kubernetes Secret object for each TLS-enabled MySQL client
```
@lichunzhu PTAL if the translation is correct.

Reply:
```suggestion
### Step 1: Create the Kubernetes Secret object for each TLS-enabled MySQL
```